cmd/trigger_ui/main.go

```diff
@@ -67,23 +67,33 @@ func (s *TriggerServer) triggerHandlerGET(c *gin.Context) {
 
 	title := meta.Get(vscommon.FieldTitle, "")
 
+	selectedAudiosource := meta.Get(vscommon.FieldExportAudioSource, "")
+
+	selectedLanguages := meta.GetArray(vscommon.FieldLangsToExport)
+
 	filenames, err := getOverlayFilenames()
 	if err != nil {
 		renderErrorPage(c, http.StatusInternalServerError, err)
 		return
 	}
 
 	c.HTML(http.StatusOK, "index.html", gin.H{
-		"title":                   title,
+		"Title":                   title,
 		"AssetExportDestinations": s.assetExportDestinations,
 		"Filenames":               filenames,
-		"Languages":               s.language,
+		"Languages":               s.languages,
+		"SelectedLanguages":       selectedLanguages,
+		"SelectedAudioSource":     selectedAudiosource,
 		"AudioSources":            s.ExportAudioSources,
 	})
 }
 
 func (s *TriggerServer) triggerHandlerPOST(c *gin.Context) {
 
+	VXID := c.Query("id")
+	Languages := c.PostFormArray("languages[]")
+	AudioSource := c.PostForm("audioSource")
```

Comment:
Watch out for variable naming. First letter upper case is usually reserved for public constants or public global variables.

Should be
```suggestion
	vxID := c.Query("id")
	languages := c.PostFormArray("languages[]")
	audioSource := c.PostForm("audioSource")
```